### DISCUSSION TITLE
ANSI Color escape Sequence bugs in GH actions output logs

### DISCUSSION THREAD

**Discussion by jpfeuffer** [community/community#48216] [Actions]
### Select Topic Area

Bug

### Body

According to the maintainers of delta, you may have bugs in your ANSI sequence handling of the GitHub Actions log output.
Please see https://github.com/dandavison/delta/issues/1310

For the full discussion.
Just try to use delta with basically any kind of diff and it will not work correctly.

Related to https://github.com/community/community/discussions/40864#discussion-4630028
And https://github.com/community/community/discussions/24976#discussioncomment-3246056 
And
https://github.com/community/community/discussions/41345
which were never replied to (again).

**Comment by ARAFATh11123** [comment]
Fhi

**Comment by jpfeuffer** [comment]
Three issues are not enough for GitHub to react?
